src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
You can use one annotation Data
```suggestion
@Data
```

Reply:
I read some topics, that were specified in the common mistakes file, about avoiding @Data annotation in Entity classes. I do not agree with all suggestions, however, I do find myself aligning with a majority of them, that's why I don't use @Data annotation here